pkg/codegen/go/gen.go

```diff
@@ -739,51 +749,72 @@ func (pkg *pkgContext) genInitFn(w io.Writer, types []*schema.ObjectType) {
 	fmt.Fprintf(w, "}\n")
 }
 
-func (pkg *pkgContext) getTypeImports(t schema.Type, imports stringSet) {
+// importBasePath converts from a repository URL to an import path.
```

Comment:
In addition to (or instead of) this function, we should add the import base path to the package-level Go info.